op-service/txmgr/txmgr.go

```diff
@@ -535,20 +537,22 @@ func (m *SimpleTxManager) increaseGasPrice(ctx context.Context, tx *types.Transa
 	gas, err := m.backend.EstimateGas(ctx, ethereum.CallMsg{
 		From:      m.cfg.From,
 		To:        rawTx.To,
-		GasFeeCap: bumpedTip,
-		GasTipCap: bumpedFee,
+		GasTipCap: bumpedTip,
+		GasFeeCap: bumpedFee,
 		Data:      rawTx.Data,
 	})
 	if err != nil {
 		// If this is a transaction resubmission, we sometimes see this outcome because the
 		// original tx can get included in a block just before the above call. In this case the
 		// error is due to the tx reverting with message "block number must be equal to next
 		// expected block number"
-		m.l.Warn("failed to re-estimate gas", "err", err, "gaslimit", tx.Gas())
+		m.l.Warn("failed to re-estimate gas", "err", err, "gaslimit", tx.Gas(),
+			"gasFeeCap", bumpedFee, "gasTipCap", bumpedTip)
 		return nil, err
 	}
 	if tx.Gas() != gas {
-		m.l.Info("re-estimated gas differs", "oldgas", tx.Gas(), "newgas", gas)
+		m.l.Info("re-estimated gas differs", "oldgas", tx.Gas(), "newgas", gas,
+			"gasFeeCap", bumpedFee, "gasTipCap", bumpedTip)
```

Comment:
Also backwards?

Reply:
yeah, fixed already while you were reviewing 😅